SpatialGDK/Source/SpatialGDK/Public/EngineClasses/SpatialWorldSettings.h

```diff
@@ -12,6 +12,38 @@
 
 #include "SpatialWorldSettings.generated.h"
 
+/**
+ * MapTestingMode allows Maps using ASpatialWorldSettings to define how Tests should run by the Automation Manager.
+ * It will handle it in a way that the current Project Settings will remain untouched.
+ */
+UENUM()
+enum class EMapTestingMode : uint8
+{
+	// It will search for ASpatialFunctionalTest, if there are any it forces Spatial otherwise Native
+	Detect				= 0,
+	// Forces Spatial to be off and use only 1 Client, the default Native behaviour
+	ForceNative			= 1,
+	// Forces Spatial to be on. Calculates the number of players needed based on the ASpatialFunctionalTests present, 1 if none exists
+	ForceSpatial		= 2,
+	// Uses current settings to run the tests
```

Comment:
nit: Uses current settings as defined by `UGeneralProjectSettings::UsesSpatialNetworking()` ?

Reply:
Not just that, all current settings including number of clients and net mode.